src/store/proposals/index.js

```diff
@@ -516,6 +541,54 @@ export default {
               ]
               proposalType = 'badge'
               break
+
+            case 'Circle' :
```

Comment:
@arsenijesavic constant instead of string `Circle`. I can see that throughout the code you use this values `Circle`, `Policy`, `Quest`... Set them as constants and reuse